docs/src/pages/components/trap-focus/trap-focus.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Trap Focus React component
+components: Unstable_TrapFocus
+---
+
+# Trap Focus
+
+<p class="description">Trap focus within a DOM node.</p>
+
+`TrapFocus` is a utility component that manage focus for its descendants.
+This is useful when implementing overlays like modal dialogs, which should not allow focus to escape them while open.
+
+When `open={true}` the trap enables, pressing <kbd>Tab</kbd> or <kbd>Shift+Tab</kbd> will circle focus within the inner focusable elements of the component.
```

Comment:
I was tempted to use `<kbd>` somewhere else but I wasn't sure how screen readers handle them?

Reply:
Nothing special in VoiceOver

<img width="965" alt="Capture d’écran 2020-08-04 à 23 43 50" src="https://user-images.githubusercontent.com/3165635/89348158-61197a00-d6ac-11ea-9258-647797434d82.png">

Reply:
Tested it in JAWS and NVDA, seemed fine.